examples/machine_learning/kmeans.cpp

```diff
@@ -112,7 +113,7 @@ int kmeans_demo(int k, bool console) {
     printf("** ArrayFire K-Means Demo (k = %d) **\n\n", k);
 
     array img =
-        loadImage(ASSETS_DIR "/examples/images/vegetable-woman.jpg", true) /
+        loadImage(ASSETS_DIR "/examples/images/spider.jpg", true) /
```

Comment:
the `isColor` is still true, this would result in an image with three channels with identical values in each channel. Can you make this `false`.

Reply:
Oops yes, thanks for catching that.